app/views/users/show.html.erb

```diff
@@ -42,6 +42,13 @@
   </section>
 <% end %>
 
-<section>
+<section id='bookmarked_segments'>
   <h1>Bookmarked Segments</h1>
+
```

Comment:
Lets add some logic here if the user does not have any bookmarks, it displays "No bookmarks yet, add some today", or something